tap/Program.cs

```diff
@@ -23,7 +23,27 @@ static void Main(string[] args)
 
                 Assembly load(string file)
                 {
-                    file = Path.Combine(appDir, file);
+                    var originalFile = file;
+                    string GetParent(string path, int level)
```

Comment:
Can you add a comment here specifying what you are trying to do? Or encapsulate this functionality in a separate method?